### PR TITLE
Button Example of full width button

### PR DESCRIPTION
**Pull request by Vineet119**

### CODE REVIEW

**Review by origami-z — COMMENTED** [review]
I believe Button will be full width when placed in a stack layout. Is there any preference on doing so? 
Worth a discussion with @lilyvc who added the button bar examples

**Comment by Vineet119** [comment]
Agreed, modified to use StackLayout as wrapper

**Comment by shey-v** [comment]
Examples look fine. However, on the site docs, we need to provide more guidance for this example.

Let's update the guidance to say:

'Use full width buttons on mobile devices or smaller viewports, the button will take up the full width of its container.'

File: site/docs/components/button/examples.mdx [hunk 1]
```diff
@@ -71,5 +71,13 @@ Use the disabled state for a button that the user can’t press.
 
 A button with the prop `disabled={true}` will suppress all functionality. If you need to allow the user to place focus on a disabled button, you also need to pass `focusableWhenDisabled={true}`.
 
+</LivePreview>
+
+<LivePreview componentName="button" exampleName="FullWidth" >
+
+## FullWidth
```
Comment (origami-z) [inline site/docs/components/button/examples.mdx:78]:
```suggestion
## Full width
```

Comment (Vineet119) [reply site/docs/components/button/examples.mdx]: corrected

**Review by shey-v — COMMENTED** [review]
'Full width' should be two words in the examples show, if we are displaying three examples and the button labels are displaying as 'primary full width button' 'secondary full width button' then I think the last example should be displayed as 'cta full width button' rather than what it is now. Otherwise, please make them all contextual.

**Review by origami-z — DISMISSED** [review]

**Comment by Vineet119** [comment]
> 'Full width' should be two words in the examples show, if we are displaying three examples and the button labels are displaying as 'primary full width button' 'secondary full width button' then I think the last example should be displayed as 'cta full width button' rather than what it is now. Otherwise, please make them all contextual.

**Review by origami-z — APPROVED** [review]

**Pull request merged**